koku/api/settings/settings.py

```diff
@@ -105,6 +107,8 @@ def __init__(self, request):
         self.factory = RequestFactory()
         self.schema = request.user.customer.schema_name
 
+        self.unleash_context = {"userId": re.sub(r"\D+", "", self.schema)}
```

Comment:
I think the goal here is to remove everything but digits from `self.schema`. Here's a simpler and more performant approach that avoids the use of regular expressions.
```suggestion
        self.unleash_context = {"userId": "".join(c for c in self.schema if c in string.digits)}
```